<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>571</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_LeftPlayerButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SWITCH PLAYER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_RightPlayerButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>251</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SWITCH PLAYER</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_CountLeft">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>191</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_CountRight">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>170</y>
       <width>191</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_announcer">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>481</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start game by pressing start button</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_left">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_right">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>140</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mode">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>390</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>MODE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_increment">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>390</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>INCREMENT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_left">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Left player game time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_right">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>110</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Right player game time</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_delay">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>390</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>DELAY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_start">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>190</y>
       <width>55</width>
       <height>54</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>▶️</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
